db/migrate/20191220183838_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[6.0]
+  def change
+    create_table :users do |t|
+      t.string :login
```

Comment:
is any of these attributes required? asking because maybe we can add a non-null constraint